bamboo-server/bamboo-server.sh

```diff
@@ -19,6 +19,14 @@ else
   echo "-> Extracting to $BAMBOO_DIR ..."
   tar xzf /tmp/atlassian-bamboo.tar.gz -C /opt
   rm -f /tmp/atlassian-bamboo.tar.gz
+  
+  echo "Add MySQL"
+  MYSQL_CONNECTOR_VERSION="5.1.38"
+  wget http://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-$MYSQL_CONNECTOR_VERSION.tar.gz -O /tmp/mysql-connector.tar.gz
+  tar xzf /tmp/mysql-connector.tar.gz -C /tmp
+  mv /tmp/mysql-connector-java-$MYSQL_CONNECTOR_VERSION/mysql-connector-java-$MYSQL_CONNECTOR_VERSION-bin.jar /opt/atlassian-bamboo-5.1/lib/
+  rm -Rf /tmp/*
```

Comment:
This is too greedy and may have unintended side effects. It can be avoided with a wrapping temporary directory.